Translations/UltraPunishments_German.lang

```diff
@@ -1,47 +1,47 @@
 #SERVER: Server
-#PUNISH: Punish
-#OR: or
-#REPORTS: Reports
-#GUI_OVERVIEW_REPORTS_ACTION: Click to show opened reports
-#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Player has no active reports
-#GUI_PLAYERVIEW_PUNISHMENTS_ACTIVE: Active Punishments
-#REPORT: Report
-#GUI_PLAYERVIEW_EXCLUDE_ACTION: Click to %1 punishments for this player
-#GUI_OVERVIEW_TEMPLATES_ACTION: Click to show templates
-#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Punishments for this player have been disabled.
-#TYPE_IN_MESSAGE: Type in a message
-#SET_A_DURATION: Click to set duration
-#PLAYERS: Players
-#REASON: Reason
-#TEMPLATES: Templates
-#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS: Active Reports by
-#EVIDENCE: Evidence
-#GUI_OVERVIEW_SETTINGS_ACTION: Click to show Settings
-#SET_A_REASON: Click to set reason
-#GUI_PLAYERVIEW_EXCLUDE_TITLE: Exclude from Punishments
-#MESSAGE: Message
-#GUI_PLAYERVIEW_PUNISHMENTS_ACTION: Click to show punishments
-#GUI_OVERVIEW_PLAYERS_ACTION: Click to show players
-#SETTINGS: Settings
-#TYPE_IN_REASON: Type in a reason
-#GUI_PLAYERVIEW_WARNINGS_ACTION: Click to show warnings
-#GUI_PLAYERVIEW_WARNINGS_NOWARNING: Player has never been warned
-#WARNINGS: Warnings
-#BAN: Ban
-#DURATION: Duration
-#GUI_OVERVIEW_TEMPLATES_MUTETEMPLATES: Mute Templates
-#GUI_PLAYERVIEW_PUNISH_LEFTACTION: **Left Click** to %1
-#PUNISHMENTS: Punishments
+#PUNISH: Bestrafen
+#OR: oder
+#REPORTS: Meldungen
+#GUI_OVERVIEW_REPORTS_ACTION: Klicke um offene Meldungen anzuzeigen
+#GUI_PLAYERVIEW_REPORTS_ACTIVEREPORTS_NONE: Spieler hat keine aktiven Bestrafungen
```

Comment:
This line should be "Spieler hat keine aktiven Meldungen" since it's "no active reports" and not punishments.